mender-auth/main.cpp

```diff
@@ -36,11 +55,29 @@ int main(int argc, char *argv[]) {
 		}
 	} else {
 		auto err = config.LoadDefaults();
-		if (error::NoError != err) {
+		if (err != error::NoError) {
 			cerr << "Failed to process command line options: " + err.String() << endl;
 			return 1;
 		}
 	}
 
+	SetLogLevel(config);
+
+	events::EventLoop loop {};
+
+	auto ipc_server {ipc::server::New(loop, config)};
+
+	const string server_url {"http://127.0.0.1:8001"};
+
+	auto err = ipc_server.Listen(server_url);
+	if (err != error::NoError) {
+		cerr << "Failed to start the listen loop" << endl;
+		cerr << "Error: " << err.message << endl;
+		return 1;
+	}
+
+	loop.Run();
+	cout << "Finished running the loop" << endl;
```

Comment:
Why not `log::info()`? (and `log::error()` above)

Reply:
No reason. Fixing